.azure-pipelines/stages.yml

```diff
@@ -567,7 +567,7 @@ stages:
     dependsOn: []
     # Skip checks if only mobile/ or docs/ have changed.
     condition: and(not(canceled()), or(succeeded(), eq(variables['PostSubmit'], true)), ne(stageDependencies.env.repo.outputs['changed.mobileOnly'], 'true'), ne(stageDependencies.env.repo.outputs['changed.docsOnly'], 'true'), ne(stageDependencies.env.repo.outputs['changed.examplesOnly'], 'true'))
-    timeoutInMinutes: 180
+    timeoutInMinutes: 240
```

Comment:
Just for my education, when do we get into a situation where we have a 4H coverage build that isn't broken?